woocommerce/wc-admin
# Task list version 2

Most of the changes for woocommerce/woocommerce#32164 

Adds a new task list design with sections under a feature flag for now, this will change to be run through an experiment.
This is currently pointed at woocommerce/woocommerce-admin#8457 PRs changes as it relies of some the logic within there.

### Accessibility




### Screenshots

<img width="871" alt="Screen Shot 2022-03-15 at 8 53 48 AM" src="https://user-images.githubusercontent.com/2240960/158372465-fef0af86-2736-479d-90ff-800f913702b8.png">

### Detailed test instructions:

-  Load this branch and edit the `includes/feature-config.php` file by enabling the `tasklist-setup-experiment-2` feature, you can also run an add_filter in your mu plugin, something like this:
```
add_filter( 'woocommerce_admin_get_feature_config', function( $feature_config ) {
	$feature_config['tasklist-setup-experiment-2'] = true;
	return $feature_config;
} );
```
You can also use[ this branch](https://github.com/woocommerce/woocommerce-admin-test-helper/pull/34) of the WooCommerce Test Helper plugin to toggle features.
- Finish the onboarding wizard and go to **WooCommerce > Home**
- Notice how there are three sections now and the first one should be opened.
- Each section when collapsed should contain a badge with the number of tasks to do.
- Copy and design should match that of the wireframes under treatment 2 - G9vi4e286yjdzyIdg8hBb1-fi-735%3A83647 (this is the easiest way to check and test).
- 'Cover the basics' section contains one task that doesn't do anything - You created <site name>, make sure that when clicking on this, it indeed doesn't do anything.
- Try to open another section, this should auto close the previous section (only one section can be open at a time)
- Step through the task list completing each step, if you completed one section it should open the next unfinished section and show a green checkmark for the collapsed completed section.
- Tasks should work as is otherwise
